addons/l10n_latam_check/models/account_chart_template.py

```diff
@@ -0,0 +1,41 @@
+from odoo import models, Command, api, _
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    @api.model
+    def _get_third_party_checks_country_codes(self):
+        """ Return the list of country codes for the countries where third party checks journals should be created
+        when installing the COA"""
+        return ["AR"]
```

Comment:
Hi @william-andre 
In this case, I think that the check module should not be dependent on l10n_ar. If the user uses checks they can be installed but maybe they don't want it to. Maybe I am wrong in my functional assumption. @MarianaLuzTorres and @jjscarafia what do you think about this?

I will wait for the answer from the functional side, in the meantime, On the technical side I have made a change in the 16.0 branch that I think resolves the problem (I think this approach maybe you will not like it, so that is why want your opinion)

1. I have left the method in the check module but returning an empty list. commit c8193cf06cb
2.  I have updated the l10n_ar module, creating there the _get_third_party_checks_country_codes, and asking if the module latam checks are installed then add the AR country codes to the existing list  951614b02e2

Reply:
> I think that the check module should not be dependent on l10n_ar.

No, but the opposite yes. But only in `master`, you shouldn't change anything in here, it was just a reminder for the forward port

Reply:
ok, we check on master

Reply:
@william-andre @jjscarafia @zaoral can we close this comment?
cc' @jco-odoo